src/services/ProjectService.jsx

```
@@ -78,12 +63,12 @@ const ProjectService = {
     try {
       fs.unlinkSync(`../banFlowProjects/${name}.json`);
     } catch (err) {
-      console.error(err);
+      /* empty */
     }
     try {
       fs.unlinkSync(`../banFlowProjects/${name}.json~`);
     } catch (err) {
-      console.error(err);
```

We need this error. We will fail silently without theses